src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -1254,36 +1260,12 @@ export class CreateVmAction extends React.Component {
                 </Tooltip>
             );
 
-        return (
-            <>
-                { createButton }
-                { this.state.showModal &&
-                <CreateVmModal
-                    mode={this.props.mode}
-                    close={this.close}
-                    networks={this.props.networks}
-                    nodeDevices={this.props.nodeDevices}
-                    nodeMaxMemory={this.props.nodeMaxMemory}
-                    // The initial resources fetching contains only ID - this will be immediately
-                    // replaced with the whole resource object but there is enough time to cause a crash if parsed here
-                    storagePools={this.props.storagePools.filter(pool => pool.name)}
-                    vms={this.props.vms}
-                    osInfoList={this.props.systemInfo.osInfoList}
-                    onAddErrorNotification={this.props.onAddErrorNotification}
-                    cloudInitSupported={this.props.cloudInitSupported}
-                    downloadOSSupported={this.props.downloadOSSupported}
-                    unattendedSupported={this.props.unattendedSupported}
-                    unattendedUserLogin={this.props.unattendedUserLogin}
-                    loggedUser={this.props.systemInfo.loggedUser} /> }
-            </>
-        );
+        return createButton;
     }
 }
 
 CreateVmAction.propTypes = {
     mode: PropTypes.string.isRequired,
-    networks: PropTypes.array.isRequired,
-    nodeDevices: PropTypes.array.isRequired,
```

Comment:
These 2 parameters are still passed CreateVmAction component in src/app.jsx

Reply:
Yes, thanks! Fixed.